vaadin-button-flow-parent/vaadin-button-flow/src/main/java/com/vaadin/flow/component/button/GeneratedVaadinButton.java

```diff
@@ -122,7 +52,10 @@
      * </p>
      *
      * @return the {@code autofocus} property from the webcomponent
+     *
+     * @deprecated since v23.3, will be removed in v24.
```

Comment:
````suggestion
     * @deprecated since v23.3, deprecated classes will be removed in v24.
```
As discussed yesterday, let's enhance all comments to note that generated classes themselves will be removed.

Reply:
Updated all comments in the latest commit